src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/Common/F1Algorithm.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP.Common;
+
+internal static class F1Algorithm
```

Comment:
Is there some paper or example implementation we could link to in doc comments for this class?